<launch>
  <arg name="bag" default="" />
  <arg name="output_file" default="$(env HOME)/autoStartGkj/command" />
  <arg name="exit_on_nonzero" default="true" />
  <arg name="topic" default="vehicleStatusMsg" />
  <arg name="queue_size" default="1" />

  <node pkg="vehicle_racing_num_ros" type="vehicle_racing_num_node" name="vehicle_racing_num_node" output="screen">
    <param name="output_file" value="$(arg output_file)" />
    <param name="exit_on_nonzero" value="$(arg exit_on_nonzero)" />
    <param name="topic" value="$(arg topic)" />
    <param name="queue_size" value="$(arg queue_size)" />
  </node>
  <node pkg="rosbag" type="play" name="rosbag_play" output="screen" args="$(arg bag)" if="$(eval arg('bag') != '')" />
</launch>
